lisa/sut_orchestrator/ready.py

```diff
@@ -18,7 +19,22 @@ def type_name(cls) -> str:
 
     @classmethod
     def supported_features(cls) -> List[Type[Feature]]:
-        return []
+        return [
+            features.Disk,
+            features.Gpu,
+            features.Nvme,
+            features.NestedVirtualization,
+            features.SerialConsole,
```

Comment:
Don't add all features, because some of them are not supported natively like Serial console, resize, security profile, start stop.

Reply:
If we don't add Serial Console, then smoke test can't run

Reply:
nvm. seems like it can't run it anyways